pkg/sriov/mocks/netlink_manager_mock.go

```diff
@@ -0,0 +1,222 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
+
```

Comment:
netlink manager interface and impl was moved to utils package

can you move this mock to be under `pkg/utils/mocks` ?

Reply:
Good catch, I moved the mock file accordingly.